src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/reported/OutcomeService.kt

```diff
@@ -103,6 +107,13 @@ class OutcomeService(
         throw ValidationException("Invalid referral transition")
     }
 
+    fun Outcome.canDelete(hasHearings: Boolean): Outcome {
```

Comment:
given we have not looked at various paths, its possible more outcome states can be removed via this api call (ie id == null), ie there are more acceptable codes to delete